Makefile

```diff
@@ -1,4 +1,4 @@
-.PHONY: help test install upgrade
+.PHONY: help test install upgrade package requirements
 
```

Comment:
Let's do this changes(if needed) in the PR with the circleci-pypi integration

Reply:
Noted.